charts/k8s-monitoring/templates/agent_config/_cluster_events.river.txt

```diff
@@ -1,7 +1,12 @@
 {{ define "agent.config.logs.cluster_events" }}
+{{- with .Values.logs.cluster_events }}
 // Cluster Events
 loki.source.kubernetes_events "cluster_events" {
   job_name   = "integrations/kubernetes/eventhandler"
+{{- if .namespaces }}
```

Comment:
This syntax `.namespaces` is it referring to a top-level value (which I couldn't find) or is it relative? Wondering why this is so short compared to `.Values.logs.cluster_events` above.

Reply:
Ah, this is a helm'ism. Or at least a go template idiom. Earlier, there's a `{{- with .Values.logs.cluster_events }}` which changes the context of `.` to be at the cluster events level. It's just useful for scoping things and not having to fully qualify every time.

Reply:
Ah ok, thanks for explaing that 👍